app/repositories/EncryptedIndividualDetailsRepository.scala

```diff
@@ -24,7 +24,7 @@ import models.individualdetails.IndividualDetailsDataCache
 import models.individualdetails.IndividualDetailsDataCache._
 import org.mongodb.scala.MongoWriteException
 import org.mongodb.scala.model._
-import org.mongodb.scala.SingleObservableFuture
+//import org.mongodb.scala.SingleObservableFuture
```

Comment:
Any particular reason, you have commented it. Removal of import would be preferrable. Normally I would comment some code which might come back later. I dont think this import would be like that.